packages/components/src/combobox-control/style.scss

```diff
@@ -0,0 +1,70 @@
+.components-combobox-control {
+	color: $dark-gray-500;
+	position: relative;
+}
+
+.components-combobox-control__label {
+	display: block;
+	margin-bottom: 5px;
+}
+
+.components-combobox-control__button {
+	border: 1px solid $dark-gray-200;
+	border-radius: 4px;
+	color: $dark-gray-500;
+	display: inline-block;
+	min-height: 30px;
+	min-width: 130px;
+	position: relative;
+	text-align: left;
+
+	&:focus {
+		border-color: $blue-medium-500;
+	}
+
+	&-input {
+		border: none;
+		height: calc(100% - 2px);
+		left: 1px;
+		padding: 0 4px;
+		position: absolute;
+		top: 1px;
+		width: calc(100% - 2px);
+	}
+
+	&-button:hover {
+		box-shadow: none !important;
```

Comment:
We should avoid using `!important` here, especially because this component is potentially reusable anywhere. I recently had to do [this](https://github.com/WordPress/gutenberg/pull/19082#discussion_r377573579) in response to the same super-specific rule; I believe @jasmussen is fixing it as we speak 😅

Reply:
Agree, we should be careful with !importants. And yes I'm working on improving the button specificity, https://github.com/WordPress/gutenberg/pull/19348/files#diff-ee8ede6238401061dd3bc7bff7d69c18L7 makes strides.

Reply:
What would be a good selector to achieve this for now then?

Reply:
I usually go with chaining the same class selector as many times as needed to override the other styles. This has the advantage of making it clear that there's a specificity issue that needs to be addressed, and it's easier to override if needed.

Reply:
I tried that with this one, but it looks like it is too hard to compete with the original selector.

Reply:
Might want to revisit this, there have been a few improvements to CSS specificity since 🙂